SmartTutor/KeycloakAuth/AllowedKeycloakRole.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SmartTutor.KeycloakAuth
+{
+    public class AllowedKeycloakRole : IAuthorizationRequirement
```

Comment:
Do we need the prefix "Allowed" for these concepts? Are there any other kinds of roles?